app/src/main/java/com/padc/nyi/moneysaver123/activities/AddIncomeActivity.java

```diff
@@ -60,10 +77,25 @@ protected void onCreate(Bundle savedInstanceState) {
         final String []dummyCategory={"aaa", "bbb"};
         ArrayAdapter adapter = new ArrayAdapter(getBaseContext(), R.layout.support_simple_spinner_dropdown_item, dummyCategory);
         adapter.setDropDownViewResource(R.layout.support_simple_spinner_dropdown_item);
-        spinnerCategory.setAdapter(adapter);
+        spinnerIncomeCategory.setAdapter(adapter);
 
         getCurrentDate();
 
+        btnIncomeSave.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                incomeVO = new IncomeVO();
+
+                incomeVO.setTitle(etIncomeTitle.getText().toString());
+                incomeVO.setAmount(Integer.parseInt(etIncomeAmount.getText().toString()));
```

Comment:
Just a suggestion, in order to reduce confusing code, you can use method overriding with a string parameter for setAmount and parseInt in that method